src/services/apiService.js

```diff
@@ -339,6 +340,12 @@ angular.module('openshiftCommonServices')
     return includeClusterScoped ? allKinds : namespacedKinds;
   };
 
+  var getPreferredVersion = function(resource) {
```

Comment:
Let's add a comment explaining what this is and why.

Reply:
Sounds good.